### PR TITLE
databricks_cli: Add version 0.215.0

### PR DESCRIPTION
**Pull request by fpvmorais**
## What type of Apps related PR is this? (check all applicable)

- [X] ✨ Add Manifest
- [ ] ♻️ Refactor
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other

## Description
Add databricks command line tool. 

> The Databricks command-line interface (also known as the Databricks CLI) provides a tool to automate the Databricks platform from your terminal, command prompt, or automation scripts.

### Add/Modify a description in the manifest?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Check URLs?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update to the latest version?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update Hashes?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Format the json manifest?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Notes in the manifest?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Application Dependencies?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Application Suggestions?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Pre-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Post-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### File persistence to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

Thanks!

### CODE REVIEW

File: bucket/databricks_cli.json
Comment (alexchapar) [inline bucket/databricks_cli.json:31]: Please take a look at terraform-ls.json to add "hash" in the autoupdate section for the hash value, which will be auto generated on each version update.

File: bucket/databricks_cli.json
Comment (alexchapar) [inline bucket/databricks_cli.json:5]: If there is no special note, you could remove it

File: bucket/databricks_cli.json
Comment (alexchapar) [inline bucket/databricks_cli.json:24]: If there is no persisted folder, you could remove this section

**Review by alexchapar — CHANGES_REQUESTED** [review]
Take a look on my comments, if you need help, dont hesitate to ask

**Comment by BotServiceAutomation** [comment]
[Your changes do not pass checks.](https://github.com/ScoopInstaller/GithubActions/wiki/Pull-Request-Checks)
### databricks_cli

- [x] Description
- [x] License
- [ ] Hashes
- [x] Checkver
- [x] Autoupdate

File: bucket/databricks_cli.json
Comment (alexchapar) [inline bucket/databricks_cli.json:31]: The "hash" segment must be within the autoupdate (next to architecture)

File: bucket/databricks_cli.json
Comment (alexchapar) [inline bucket/databricks_cli.json:18]: The shortcut must contains two values : the exe and the "display name". you can take a look at the vscode.json.

That said, if it's only a CLI, it might not be relevant to have this in the windows menu (within scoop)... I'll let you the judge of that ;)

[Sorry for earlier, I didn't see this little issue while reviewing it on my phone ;)]

**Review by alexchapar — CHANGES_REQUESTED** [review]
After your changes, if you still get issue with the hash update, I will take a better look... but for now i think it's the issue...

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### databricks_cli

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by alexchapar — APPROVED** [review]
Thanks for this new manifest :)

**Comment by fpvmorais** [comment]
Sorry for all the missed stuff :)

**Pull request merged**
